hail/python/hail/experimental/db.py

```diff
@@ -0,0 +1,55 @@
+from ..matrixtable import MatrixTable
+import requests
+import json
+import hail as hl
+
+class DB:
+    
+    _annotation_dataset_urls = None
+    
+    @staticmethod
+    def annotation_dataset_urls():
+        if DB._annotation_dataset_urls is None:
+            r = requests.get("http://storage.googleapis.com/hail-datasets/datasets.json")
+            j = r.json()
+            
+            DB._annotation_dataset_urls = {(x["name"], x["reference_genome"]): (x["path"], x["gene_key"]) for x in j}
+        
+        return DB._annotation_dataset_urls
+    
+    def annotate_rows_db(self,mt,*names):
+        """
+            Examples
+            --------
+            Annotates rows based on keyword implementation of annotation name. The user can type in multiple annotation names when attaching to their datasets.
```

Comment:
In our docs, we aim for fewer than 80 characters per line because some people view the docs in their terminal (for example, start `ipython` a type this:
```
import hail as hl
?hl.init
```
Hitting enter after `?hl.init`.)

Let's add new lines so this long line is no more than 80 characters. A lot of text editors have a formatting tool that will do this for you. I'm not sure how to do it in sublime.